git/libgit2/transport/http.go

```diff
@@ -238,10 +238,8 @@ func (t *httpSmartSubtransport) Action(transportOptionsURL string, action git2go
 	}
 
 	stream := newManagedHttpStream(t, req, client)
-	if req.Method == http.MethodPost {
-		stream.recvReply.Add(1)
-		stream.sendRequestBackground()
-	}
+	stream.recvReply.Add(1)
+	stream.sendRequestBackground()
```

Comment:
I am not entirely sure why upstream was handling this only on `POST`.
But I probably would wait until `pkg` is fully integrated across the controllers before making this change.